e2e/_suites/helm/README.md

```diff
@@ -0,0 +1,91 @@
+[![Build Status](https://beats-ci.elastic.co/buildStatus/icon?job=e2e-tests%2Fe2e-testing-helm-daily)](https://beats-ci.elastic.co/job/e2e-tests/job/e2e-testing-helm-daily/)
```

Comment:
As long as the daily job does not wait for the downstream job's status then let's use the downstream one:

```suggestion
[![Build Status](https://beats-ci.elastic.co/buildStatus/icon?job=e2e-tests%2Fe2e-testing-mbp%2Fmaster)](https://beats-ci.elastic.co/job/e2e-tests/job/e2e-testing-mbp/job/master/)
```